third_party/2/concurrent/futures/__init__.pyi

```diff
@@ -32,15 +32,15 @@ class Executor:
     def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> bool: ...
 
 class ThreadPoolExecutor(Executor):
-    def __init__(self, max_workers: Optional[int] = ...) -> None: ...
+    def __init__(self, max_workers: int) -> None: ...
```

Comment:
The old code looks correct to me: https://github.com/agronholm/pythonfutures/blob/master/concurrent/futures/thread.py#L98.